tasks/client.yml

```diff
@@ -2,3 +2,44 @@
 # TODO Install clients:
 ## kubectl
 ## aws-iam-authenticator
+
+- name: "Get kubectl path"
+  shell: "which kubectl || echo 'not found'"
+  register: kubectl_path
+
+- name: Create bin directory
+  file:
+    path: /opt/k8s
+    state: directory
```

Comment:
@brunomanzo the directory `HOME/bin` is recommended. `/opt` should have extra permissions that is not the scope of this role.